hack/after-build/run-codegen.sh

```diff
@@ -33,6 +34,7 @@ clientgen=$(kube::util::find-binary "client-gen")
 # update- and verify- scripts.
 ${clientgen} "$@"
 ${clientgen} -t "$@"
+${deepcopygen} "$@"
```

Comment:
I gather the unit test was failing by reading the previous conversations. May I ask what changes did you make to let the Jenkins unit test pass?